templates/build/news-feed-loop.build.hbs

```diff
@@ -39,10 +39,10 @@
           \{{/if}}
         </div>
         \{{#if [Title]}}
-        <h2 class="news-feed-item-title">\{{[Title]}}</h2>
+        <h2 class="news-feed-item-title" data-line-clamp="3">\{{[Title]}}</h2>
```

Comment:
The CSS you removed uses the following selector.

```css
.new-news-feed-list-container .news-feed-list-item:not(.open) .slide-over .news-feed-item-inner-content .news-feed-item-description
```

This suggests only the description needs to be clamped to 3 lines, not the title.

Reply:
@tonytlwu Yes, but I thought that it would be better do make similar to the simple list title.
Should I remove it then?

Reply:
Leave it as it is then.